omero/sysadmins/dropbox.txt

```diff
@@ -41,13 +41,16 @@ the following more specific requirements:
 
 .. note:: It is likely, but not guaranteed, that DropBox will run on later
     versions of Windows, however only tested versions are supported by default.
-    Setting the property :property:`omero.fs.platformCheck` to ``False`` in
+    
+    Setting the property :property:`omero.fs.platformCheck` to ``False``
+    in,
+    
```

Comment:
Will had suggested that this space was not rendered properly, so I was just trying to highlight the space by reorganising stuff. But I guess it could be improved. Will look into other options.